src/equations/passive_tracers.jl

```diff
@@ -0,0 +1,222 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+@doc raw"""
+    PassiveTracerEquations(flow_equations)
+
+Adds passive tracers to the `flow_equations`. The tracers are advected by the flow velocity.
+These work for arbitrary NDIMS with arbitrary numbers of tracers NTracers. For one dimension, with 
+one tracer ``\xi`` and flow with density and velocity ``\rho, v`` respectively, the equation of the
```

Comment:
I am medium-happy with using $\xi$ as this is more commonly used in coordinate transformations. What about something else like $\chi$ (`chi`)? This variable name is at least not used right now in the code at all :)